packages/fxa-auth-server/fxa-oauth-server/lib/jwt_sub.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const hex = require('buf').to.hex;
+const hkdf = require('../../lib/crypto/hkdf');
+const config = require('./config');
+
+const PPID_ENABLED = config.get('ppid.enabled');
+const PPID_CLIENT_IDS = new Set(config.get('ppid.enabledClientIds'));
+const PPID_ROTATING_CLIENT_IDS = new Set(config.get('ppid.rotatingClientIds'));
```

Comment:
Do you even use this?

Reply:
https://github.com/mozilla/fxa/pull/1679/files#diff-8e82b39c55246a507d7c2a3fe5dc263aR30